python/taichi/main.py

```diff
@@ -69,6 +70,58 @@ def test_cpp(args):
     return int(task.run(*test_files))
 
 
+def get_benchmark_baseline_dir():
+    import taichi as ti
+    return os.path.join(ti.core.get_repo_dir(), 'baselines')
```

Comment:
BTW, can we place these data not in the root directories? (I don't think these data deserve a slot in the root dir..)

E.g. maybe `benchmark/data/baseline`? Also, I'd suggest using the singular form, i.e. `baseline` and `output`

Reply:
As we already have the `benchmarks` folder, I agree to put the files in the folder.

We have many folders with plural form names. Is using singular form a better coding style?

Reply:
> We have many folders with plural form names. Is using singular form a better coding style?

Feels a bit weird to see `outputs`, but this isn't a big deal.. I think the important part is to not put too much under the root directory